index.html.1

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html>
+<head>
+<title>Index of /pub/slackware-iso/slackware64-15.0-iso on ftp.slackware.com:21</title>
```

Comment:
I somehow doubt that adding this `index.html.1` was intentional ;-)

Can you please remove it again (with another PR)?